ll.tm

```diff
@@ -5010,11 +5014,12 @@ ParamAttr -> ParamAttr
 	| 'readonly'
 	| 'returned'
 	| 'signext'
-	| 'sret'
 	| 'swifterror'
 	| 'swiftself'
 	| 'writeonly'
 	| 'zeroext'
+	| 'noprofile'
```

Comment:
Same as in https://github.com/llir/grammar/pull/21#pullrequestreview-671861182, neither `noprofile` nor `mustprogress` are `ParamAttrs`.

From `parseOptionalParamAttrs`:

```cpp
    case lltok::kw_mustprogress:
    // ...
    case lltok::kw_noprofile:
    // ...
      HaveError |=
          error(Lex.getLoc(), "invalid use of function-only attribute");
      break;
```